### PR TITLE
modernize

### PR DESCRIPTION
**Pull request by grosser**
this guy only worked with rails 2 and was tests on rails 2 ... time for an upgrade ...

@jcheatham @steved555 @staugaard

### CODE REVIEW

**Comment by grosser** [comment]
fyi @pschambacher

File: action_mailer-logged_smtp_delivery.gemspec [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
+  gem.files         = Dir.glob('lib/**/*') + ['README.md']
 
   gem.name          = 'action_mailer-logged_smtp_delivery'
-  gem.require_paths = ['lib']
   gem.version       = '1.1.0'
+  gem.license       = "Apache V2"
 
-  gem.add_development_dependency 'actionmailer', '2.3.16'
-  gem.add_development_dependency 'minitest'
+  gem.add_development_dependency 'actionmailer', '~> 3.2.19'
```
Comment (pschambacher) [inline action_mailer-logged_smtp_delivery.gemspec:13]: What about using `> 3.2.19` so we don't block a possible migration to Rails 4 ? :smile:

**Comment by pschambacher** [comment]
:+1: good bye rails 2

File: action_mailer-logged_smtp_delivery.gemspec (hunk 1 above)
Comment (grosser) [inline action_mailer-logged_smtp_delivery.gemspec:13]: I'll do a separate test for newer rails versions, but block untested
upgrades so we do not do untested updates again, was pretty hacky and
changed the log format

Comment (pschambacher) [inline action_mailer-logged_smtp_delivery.gemspec:13]: OK

**Comment by grosser** [comment]
using zendesk repo now

**Pull request closed without merging**